chcemvediet/tests/test_views.py

```diff
@@ -1,40 +1,21 @@
 # vim: expandtab
 # -*- coding: utf-8 -*-
-import unittest
-
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 
 from poleno.utils.test import ViewTestCaseMixin
 
-class IndexViewTest(ViewTestCaseMixin, TestCase):
```

Comment:
@martinmacko47 `'index'` sa pretransformoval na `'homepage'`. Ma aj `'about'` view svoju nahradu? Je to `sk/o-projekte/`, ktory pouziva template `about.html`, vsak?

Reply:
@viliambalaz Nie, `about` netreba, pretoze ten je implementovany cez pages app. Tu testujeme iba viewy, ktore su implementovane priamo v `chcemvediet/views.py`. Cize okrem `homepage` by sa zislo otestovat ja view `search`.

Reply:
:ok: